### DISCUSSION TITLE
Mucommander should also be a terminal command

### DISCUSSION THREAD

**Discussion by piotrek-k** [mucommander/mucommander#1092] [Ideas]
On Linux it is quite common to launch file manager from withing terminal session. So you navigate in terminal to some directory, decide to use file manager from there, so you type e.g. `mu` in terminal and mucommander launches in this directory. Should work similarly to how MidnightCommander's `mc .` command works.

**Comment by pskowronek** [comment]
I like the idea!
It can be done even now tho, just create a simple command or define `alias`. 

Would you be able to provide PR with PoC (GitX could be some inspiration, macOS only, but I've similar functionality in other projects) :
![obraz](https://github.com/mucommander/mucommander/assets/360665/780f3a2f-2b7a-4e78-b7eb-e36c09ec86dc)

**Comment by ahadas** [comment]
> ... Should work similarly to how MidnightCommander's `mc .` command works.

I'm not quite sure what's missing then - we install muCommander in /opt [which is the default of jpackage](https://docs.oracle.com/en/java/javase/17/jpackage/packaging-tool-user-guide.pdf) so it's probably missing in your PATH but once /opt/mucommander/bin/muCommander is added to the PATH, you can then achieve the same functionality by running `muCommander .` (or `muCommander . .` if you want the current directory to appear in both panels).
